src/bin/help.test.ts

```diff
@@ -0,0 +1,27 @@
+import chalk from "chalk";
+import { SpyInstance, beforeEach, describe, expect, it, vi } from "vitest";
+
+import { logHelpText } from "./help.js";
+
+let mockConsoleLog: SpyInstance;
+
+describe("logHelpText", () => {
+	beforeEach(() => {
+		mockConsoleLog = vi
+			.spyOn(console, "log")
+			.mockImplementation(() => undefined);
+	});
+
+	it("logs help text when called", () => {
+		logHelpText([
+			chalk.yellow(
+				"⚠️ This template is early stage, opinionated, and not endorsed by the TypeScript team. ⚠️",
+			),
+			chalk.yellow(
+				"⚠️ If any tooling it sets displeases you, you can always remove that portion manually. ⚠️",
+			),
+		]);
+
+		expect(mockConsoleLog.mock.calls).toMatchSnapshot();
```

Comment:
Just a heads up, the CI fails because chalk's colors are different on GitHub than locally. https://github.com/JoshuaKGoldberg/expect-no-axe-violations/pull/3 is another place I've had to work around it. I don't like that solution though, it's annoying making even more root files & Vitest config.

You can work around it with code like this instead that replaces chalk's with functions with ones that just return strings:

```ts
function makeProxy<T extends object>(receiver: T): T {
	return new Proxy(receiver, {
		get: () => makeProxy((input: string) => input),
	});
}

vi.mock("chalk", () => ({
	default: makeProxy({}),
}));
```

I have on my todo-list making a dedicated package like this.

Reply:
Thank you! I've added this code.